instructor/decorators.py

```diff
@@ -0,0 +1,71 @@
+from typing import Callable, Any, TypeVar
```

Comment:
lets rename this to validators 
then also bring it to the main namespace in __init__.py

Reply:
Renamed and added to the main namespace